optuna/visualization.py

```diff
@@ -51,7 +51,7 @@ def is_available():
 
 
 def plot_intermediate_values(study):
-    # type: (Study) -> None
+    # type: (Study) -> Figure
```

Comment:
This is the issue on existing code, but I think we can delete the access to the private module `plotly.graph_objs._figure` at [L26](https://github.com/optuna/optuna/pull/704/files#diff-5f61268549ae81f3dd0bd1f4b97a5a74R26) by replacing `Figure` in type hints to `go.Figure` as follows:
```suggestion
    # type: (Study) -> go.Figure
```